src/JustEat.StatsD/Buffered/Tags/StatsDTagsFormatter.cs

```diff
@@ -45,27 +45,27 @@ public virtual int GetTagsBufferSize(in IDictionary<string, string?>? tags)
             }
 
             return _prefix.Length
-                + Encoding.UTF8.GetByteCount(GetFormattedTags(tags!))
+                + Encoding.UTF8.GetByteCount(this.FormatTags(tags!))
                 + _suffix.Length;
         }
         
         /// <inheritdoc />
-        public virtual string GetFormattedTags(in IDictionary<string, string?>? tags)
+        public virtual string FormatTags(in IDictionary<string, string?>? tags)
         {
             if (AreTagsPresent(tags))
             {
-                return _prefix + GetFormattedTags(tags!) + _suffix;
+                return _prefix + this.FormatTags(tags!) + _suffix;
             }
 
             return string.Empty;
         }
 
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
-        private string GetFormattedTags(IDictionary<string, string?> tags) =>
-            string.Join(_tagsSeparator,tags.Select(tag => GetFormattedTag(tag)));
+        private string FormatTags(IDictionary<string, string?> tags) =>
+            string.Join(_tagsSeparator,tags.Select(tag => this.FormatTags(tag)));
```

Comment:
I think you can save the Lambda here and just point directly to the method?

```suggestion
            string.Join(_tagsSeparator, tags.Select(FormatTags));
```

Reply:
Done!